src/Text/Ginger/Parse.hs

```diff
@@ -1170,6 +1171,19 @@ filterP = do
     args <- option [] $ groupP "(" ")" funcArgP
     return $ \e -> CallE pos func ((Nothing, e):args)
 
+testExprP :: Monad m => Parser m ((Expression SourcePos) -> (Expression SourcePos))
+testExprP = do
+    pos <- getPosition
+    keyword "is"
+    spacesOrComment
+    funcName <- atomicExprP--stringLiteralExprP
+    args <- option [] $ groupP "(" ")" funcArgP
```

Comment:
We also need to support parens-less argument syntax: `{% if x is lt 1 %}` (is equivalent to `{% if x is lt(1) %}`).